src/TimeService/timeservice-api.c

```diff
@@ -0,0 +1,85 @@
+#include "timeservice-api.h"
+
+static uint64_t starting_time;
+
+void nanoseconds_to_cuc_timestamp(uint64_t nanoseconds, CUCTimestamp *timestamp)
```

Comment:
Add TimeService context to make code pseudo objective

Reply:
Done